pkg/telemetry/telemetry.go

```diff
@@ -119,14 +119,14 @@ func (t *Telemetry) GetData() Data {
 
 // Send telemetry information to SWA
 func (t *Telemetry) Send() {
-	// always log step telemetry data to logfile used for internal use-case
-	t.logStepTelemetryData()
-
 	// skip if telemetry is disabled
 	if t.disabled {
 		return
 	}
 
+	// always log step telemetry data to logfile used for internal use-case
```

Comment:
We need this logging also if telemetry is disabled! See the comment. Please move it back to before the if t.disabled {} is called

Reply:
Can we then introduce another parameter for people who even don't want this to be logged?

Reply:
I need to discuss the details with @OliverNocon as it touches internal use-cases where we **need** the logging to be activated.

Reply:
Lets take this discussion offline. For now I've just reverted this part.